zokrates_cli/examples/book/declaration.zok

```diff
@@ -0,0 +1,6 @@
+def main() -> ():
+	// declare and define a
```

Comment:
should say `// declare and define my_variable`, or variable should be called `a`

Reply:
good catch!